operations/mimir-mixin/dashboards/overview.libsonnet

```diff
@@ -33,6 +33,7 @@ local filename = 'mimir-overview.json';
     assert std.md5(filename) == 'ffcd83628d7d4b5a03d1cafd159e6c9c' : 'UID of the dashboard has changed, please update references to dashboard.';
     ($.dashboard('Overview') + { uid: std.md5(filename) })
     .addClusterSelectorTemplates()
+    .addShowNativeLatencyVariable()
```

Comment:
One dashboard says 'Latency metrics: classic/native' and the other says 'Show historic data: hide/show'. Could we standardise this? If we use the second one, maybe 'Show historic data: yes/no' or 'Historic data: hide/show'?

Reply:
Nvm, out of scope for this PR.